tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -69,6 +69,8 @@ def create_parser(product_choices=None):
                         help="Number of simultaneous processes to use")
     parser.add_argument("--max-restarts", type=int, default=5,
                         help="Maximum number of browser restart retries")
+    parser.add_argument("--max-restart-backoff", type=int, default=0,
+                        help="The maximum wait between retrying browser restarts; zero for no backoff")
```

Comment:
I'd assume seconds if I was using the script, but it's probably good to explicitly define the unit here